app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_comment_counter
+
+  def update_coumment_counter
+    Post.find(post_id).update(comments_counter: Comment.where(post_id).count)
+  end
```

Comment:
- [ ] Great job for making use of one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html), `after_save`. it is also good to make this method `private`. Kindly make it private. It should look something similar to this :point_down: 

```
  class Comment < ApplicationRecord
  belongs_to :author, class_name: 'User'
  belongs_to :post

  after_save :update_comment_counter

  private
  def update_coumment_counter
    Post.find(post_id).update(comments_counter: Comment.where(post_id).count)
  end
```